cleanup_utils/openstack_cleanup.py

```diff
@@ -131,9 +131,15 @@ def run_cleanup(self):
                 if not self.dry_run:
                     instance.delete()
 
-            else:
+            elif instance_age:
                 logger.info(
                     "    * SKIPPING: Instance is only %s seconds old (age threshold is %s seconds).",
                     instance_age,
                     self.age_limit
                 )
+            else:
+                logger.info(
+                    "    * SKIPPING: Couldn't parse instance age.",
```

Comment:
@giovannicimolin, it would be better use format strings to include the variables in the message or not include them at all since we know that `instance_age` is going to be `None`.